certbot/cli.py

```diff
@@ -687,6 +688,19 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
              "multiple -d flags or enter a comma separated list of domains "
              "as a parameter.")
     helpful.add(
+        [None, "run", "certonly"],
+        "--cert-name", dest="certname",
+        metavar="CERTNAME", default=None,
+        help="Certificate name to apply. Only one certificate name can be used "
+             "per Certbot run. Show certificate names by running certificates "
```

Comment:
Yes and maybe also better to say something like `by running "certbot certificates"` or ` by running cerbot with the "certificates" subcommand`?